src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -800,6 +800,14 @@ - (void)removeDeviceControllerDelegate:(id<MTRDeviceControllerDelegate>)delegate
     }
 }
 
+- (void)clearDeviceControllerDelegates
```

Comment:
Does the _XPC version of this need this too?

Reply:
It should be called from the base class now.